includes/html/pages/ports/list.inc.php

```diff
@@ -33,9 +33,9 @@
         <table id="ports" class="table table-condensed table-hover table-striped">
             <thead>
             <tr>
-                <th data-column-id="device" data-formatter="device">Device</th>
-                <th data-column-id="port"<?php echo $sort ?>>Port</th>
-                <th data-column-id="ifLastChange" data-converter="duration">Status Changed</th>
+                <th data-column-id="hostname" data-formatter="device">Device</th>
+                <th data-column-id="ifDescr"<?php echo $sort ?> data-formatter="device">Port</th>
```

Comment:
```suggestion
                <th data-column-id="ifDescr"<?php echo $sort ?> data-formatter="port">Port</th>
```

?

Reply:
yeah, changed it to a formatter in one of the last commits